src/screens/VODScreen/index.js

```diff
@@ -124,6 +128,23 @@ export default class VODScreen extends BaseScreen {
     })
   }
 
+  
+
+  _handleLeftRelease(){
+    this.currentposition = this.currentposition-(1000*60*2);
+    player.jump(this.currentposition);
+    this.$mediaplayerProgress(this.currentposition, this.movieduration)
+  }
+
+
+  _handleRightRelease(){
+    this.currentposition = this.currentposition+(1000*60*2);
```

Comment:
Please define constants instead of magic numbers

Reply:
constants.MOVE_POSITION added into constants .js and start to using in VODScreen/index.js